src/pages/trade/HorizontalPanel.tsx

```diff
@@ -308,13 +309,16 @@ export const HorizontalPanel = ({ isOpen = true, setIsOpen }: ElementProps) => {
 const $AssetIcon = styled(AssetIcon)`
   font-size: 1.5em;
 `;
+const collapsibleTabsType = getSimpleStyledOutputType(CollapsibleTabs, {} as {});
```

Comment:
`{} as {}` is it possible to make this optional w/ a default in `getSimpleStyledOutputTypes`

Reply:
Oh yeah I like that. Added!